test/JDBC/upgrade/14_12/schedule

```diff
@@ -445,3 +445,4 @@ test_conv_string_to_datetimeoffset-before-15_7
 test_conv_string_to_smalldatetime
 test_conv_string_to_time-before-15_7
 BABEL-3401
+babel_726
```

Comment:
shouldn't we add babel_726-before-14_12... file entry to older version schedule files?